internal/commands/result.go

```diff
@@ -791,13 +791,22 @@ func writeConsoleSummary(summary *wrappers.ResultSummary, featureFlagsWrapper wr
 }
 
 func printPoliciesSummary(summary *wrappers.ResultSummary) {
-	fmt.Printf(tableLine + "\n")
-	if summary.Policies.BreakBuild {
-		fmt.Printf("            Policy Management Violation - Break Build Enabled:                     \n")
-	} else {
-		fmt.Printf("            Policy Management Violation:                     \n")
-	}
+	hasViolations := false
 	if len(summary.Policies.Policies) > 0 {
```

Comment:
Why is this if necessary?
if len(summary.Policies.Policies) == 0, we will not iterate it in the for loop either way

Reply:
This is the bug, the function get a policy without a roll violation.